scripts/setup/s3_storage.py

```diff
@@ -1,12 +1,11 @@
 import logging
-import requests
+import secrets
```

Comment:
I think this will need to be `import setup.secrets as secrets`?

Reply:
I believe so. I had to update the stripe connector to that style. I'll update this file now.